MakeCitron.Makefile

```diff
@@ -1,7 +1,7 @@
 include config.Makefile
 -include config.custom.Makefile
 
-BASEVERSION ?= v2
+BASEVERSION ?= v3
```

Comment:
Will conflict with already existing v3 branch? I expected something like `v3-legacy`

Reply:
Mob review: it's just an example.